ssl/statem/statem.c

```diff
@@ -937,3 +937,12 @@ int ossl_statem_app_data_allowed(SSL *s)
 
     return 0;
 }
+
+/*
+ * This function returns 1 if TLS exporter is ready to export keying
+ * material, or 0 if otherwise.
+ */
+int ossl_statem_export_allowed(SSL *s) {
+    return s->s3->previous_server_finished_len != 0 &&
```

Comment:
I would prefer if you put the && at the start of the next line, since you're already editing this function.  but not required.